controller/src/test/java/io/pravega/controller/rest/v1/PingTest.java

```diff
@@ -63,8 +69,57 @@ public void tearDown() {
 
     @Test
     public void test() {
-        String streamResourceURI = "http://localhost:" + serverConfig.getPort() + "/ping";
+        String streamResourceURI = getURLScheme() + "://localhost:" + serverConfig.getPort() + "/ping";
```

Comment:
Could we maybe use `javax.ws.rs.core.UriBuilder` or something?

Reply:
Using `UriBuilder`. Fixed.